public/assets/sass/footer.scss

```diff
@@ -0,0 +1,44 @@
+@import 'util/colours';
+
+.footer {
```

Comment:
This is really neat how all the code in the file is encapsulated within `.footer`, keeping the styles of elements within the footer from conflicting with anything outside of it.  Would it be possible to replicate this on the other files? (e.g. putting all navbar styles within the `.navbar` class, carousel within `.carousel`, etc.)?

Reply:
resolved in new commit :)